__tests__/app.test.js

```diff
@@ -112,14 +112,88 @@ describe("Get /api/articles", () => {
       );
     });
   });
-  test("article objects are served sorted by date in descending order", async () => {
+  test("200: article objects by default are served sorted by date in descending order", async () => {
     const {
       body: { articles },
     } = await request(app).get("/api/articles").expect(200);
     expect(articles).toBeSortedBy("created_at", {
       descending: true,
     });
   });
+  test("200: article objects can be served sorted by date in ascending order", async () => {
+    const {
+      body: { articles },
+    } = await request(app).get("/api/articles?order=asc").expect(200);
+    expect(articles).toBeSortedBy("created_at", {
+      ascending: true,
```

Comment:
Jest Sorted doesn't take `ascending` as an option. This test still passes because `descending` defaults to false - therefore, it would be worth changing this to show a potential employer that you understand how to use Jest Sorted